### ISSUE TITLE
[AVM Module Issue]: Variable zone_balancing_enabled not recognized

### ISSUE THREAD

**Issue by stredovek** [Azure/terraform-azurerm-avm-res-web-site#41]
### Check for previous/existing GitHub issues

- [X] I have checked for previous/existing GitHub issues

### Issue Type?

Bug

### Description

[new_service_plan](https://registry.terraform.io/modules/Azure/avm-res-web-site/azurerm/latest#input_new_service_plan)  does not contain variable definition for zone_balancing_enabled so it is not possible create region redundant FA. https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/service_plan#zone_balancing_enabled

**Comment by donovm4** [comment]
@stredovek - I am currently not able to test the functionality of the `zone_balancing_enabled` but will add the argument to the `new_service_plan` variable.

**Comment by stredovek** [comment]
@donovm4  I have tested it successfully with following arguments in  `azurerm_service_plan`:
``` terraform
  zone_balancing_enabled       = true
  maximum_elastic_worker_count = 3
  worker_count                 = 3
```

**Comment by donovm4** [comment]
Ok - thank you @stredovek for your insight. I will spin up another PR to add the following arguments as well, and then close out this issue:

- `maximum_elastic_worker_count`
- `worker_count`
- `per_site_scaling_enabled`

**Comment by donovm4** [comment]
bug fix addressed in v0.4.0 release
